opencti-platform/opencti-front/src/components/TextField.jsx

```diff
@@ -40,9 +40,12 @@ const TextField = (props) => {
     [onSubmit, setTouched, name],
   );
   const [, meta] = useField(name);
+  const { value, ...otherProps } = fieldToTextField(props);
+
   return (
     <MuiTextField
-      {...fieldToTextField(props)}
+      {...otherProps}
+      value={value ?? ''}
```

Comment:
Reason: `value` can be null. But an input should not be initialized with null or undefined value, instead use an empty string. By so, React can properly track changes with the input.